css-canvas-medical/styles.css

```diff
@@ -0,0 +1,223 @@
+
+.container {
+  max-width: 1300px;
+  margin: auto;
+}
+
+.column-third {
+  flex-basis: 30%;
+}
+
+.column-two-thirds {
+  flex-basis: 66%;
+}
+
+.row {
+  display: flex;
+  margin: auto;
+  margin-left: 3rem;
+  margin-right: 3rem;
+}
+
+.align-center {
+  align-items: center;
+}
+
+.column-full {
+  flex-basis: 100%;
+}
+
+.justify-between {
+  justify-content: space-between;
+}
+
+.two-rem-margins {
+  margin: 2rem;
+}
+
+.two-rem-padding {
+  padding: 3rem;
+}
+
+.large-font {
+  font-size: 48px;
+}
+
+.padding {
+  padding: 4rem 0 1rem 6.5rem;
+}
+
+.border {
+  border-bottom: solid 1px #e7e9ec;
+}
+
+button {
+  font-family: 'Avenir', sans-serif;
+  border-color: #304464;
+  color: white;
+  background-color: #304464;
+  padding: 1rem 1rem;
+  border-radius: 0.25rem;
+  border: none;
+  width: 16rem;
+  height: 4rem;
+}
+
+.get-started {
+  font-size: 25px;
+}
+
+.picture-heading {
+  font-family: 'Tiempos Headline', sans-serif;
+  font-size: 48px;
+  font-weight: normal;
+}
+
+.p-style {
+  font-size: 21px;
+  line-height: 37px;
+}
+
+.overlay {
+  background: linear-gradient(rgba(0, 0, 0, 0.25), rgba(0, 0, 0, 0.25));
+  position: absolute;
+  width: 1207px;
+  height: 680px;
+}
+
+.back-image {
+  display: flex;
+  background-image: url('images/hero.jpg');
+  width: 1400px;
+  height: 740px;
+  background-size: 100%;
+  background-repeat: no-repeat;
+  position: relative;
+  opacity: 90%;
+}
+
+.text-on-image {
+  font-family: 'Avenir', sans-serif;
+  position: absolute;
+  color: white;
+  margin-right: 47rem;
```

Comment:
That's a LOT of margin 🤔 

Can you do this with the box offset properties since you are using positioning? If you set the parent element as `relative`, it should be relatively easy to move where you want it to be.

Reply:
![Screen Shot 2021-07-05 at 1 41 31 PM](https://user-images.githubusercontent.com/52221756/124516713-25dd6a80-dd97-11eb-8957-c8ef3ee3798d.png)